pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-chatops"
-version = "1.3.1"
+version = "1.4.0"
```

Comment:
As an alternative to backwards-compatibility as I'm suggesting in my other changes, if you don't want to maintain backwards compatibility, this would need to become `2.0.0` per semantic versioning.

Reply:
I like your thoughts on backwards compatibility, so I'll leave the version as is.  I think we can make 2.0.0, whenever that is, the change that breaks backwards compatibility for Webex.